### ISSUE TITLE
Statistical Inference: Display uploaded data 

### ISSUE THREAD

**Issue by ashokkrish** [ashokkrish/CougarStats#35]
TBA

**Comment by bryce-carson** [comment]
I found the _perfect_ solution to this in [DataEditR](https://dillonhammill.github.io/DataEditR/articles/DataEditR.html). It's [a Shiny module](https://mastering-shiny.org/scaling-modules.html), so we hardly need to integrate it, but it'll still take several hours at least.

https://dillonhammill.github.io/DataEditR/articles/DataEditR.html

**Comment by bryce-carson** [comment]
The idea is to use this _singular_ interface for uploading and editing data, and when we request the data from this module we can use it across the rest of the application.

**Comment by bryce-carson** [comment]
I'll look for a good place to put it, but for now I'm going to finish #33.

If you want other things prioritized, the #5 for example, we can work on that next week. When I'm waiting for any turn-around whilst working on #5 with you next week I'll make small efforts to integrate this and clean up the related code in CougarStats.

**Comment by bryce-carson** [comment]
[DataEditR](https://dillonhammill.github.io/DataEditR/articles/DataEditR.html) is not perfect, actually. A better experience is found in [datamods](https://dreamrs.github.io/datamods/), but it is not working _perfectly_ yet.

**Comment by bryce-carson** [comment]
> @bryce-carson > _* module in the data upload area of the app_
> 
> I would like this module **only to be activated** in the Simple Linear Regression (SLR)/ Multiple Linear Regression (MLR) context. I see great positives and potential for it to come under the descriptive stats and statistical inference but just not now.

Yes, I noted that immediately after our discussion today. I'll try to find a workflow that appears nice and works well. _datamods_ (data modules) is stable, as far as I can tell, and has had updates recently so it isn't old software or stuff (shinyDarkmode is actually quite old and I imagine not the most robust solution; modern Shiny [what we're using] has built-in support for dark mode [there's a built-in input widget for it, actually!]).

I'll limit this to the Regression and Correlation area of the application for now; which is already the case. I'll remove the navigation bar tab for it and find a nice looking way to enter and exit the data editor. I think I'll use a keyboard shortcut, given we're only targeting desktop with this application for the moment. That'd work well.

**Comment by bryce-carson** [comment]
@ashokkrish, I'm labelling this isuse "wontfix" because it _shouldn't_ have a solution for just itself.

As discussed, the best way forward is a centralized place for the application as a whole to have data upload and manipulation (this will also reduce development effort because of existing open source solutions; similarly to the react-chart-editor we adopted in Episim).

I won't close the issue; the label is mostly for myself, and a reminder for you of this comment and the discussion surrounding the application's future.
